### DISCUSSION TITLE
If you use Hidden mod without fading multiplier, how many multiplier do you want?

### DISCUSSION THREAD

**Discussion by tematikkp** [ppy/osu#20699] [Polls]
What will you choose?

My opinion is 1.01x if you use Hidden mod with only not showing the approach circles, as it is for sightreaders, and they might misread it.

**Comment by okolenmi** [comment]
1.05x because it's much harder than common HD. Especially with small AR... With common HD you know that it's almost time to click when circle disappeared. Without this useful thing accuracy will rapidly fall. No accuracy = -30%+-pp

**Comment by Doksha** [comment]
I dont believe peppy wants any of the mods to have different multipliers based on settings, either the default multiplier or nothing.
